src/renderer/components/kube-object/kube-object-details.tsx

```diff
@@ -36,18 +46,18 @@ export function hideDetails() {
   showDetails();
 }
 
-export function getDetailsUrl(details: string, resetSelected = false) {
-  const detailsUrl = kubeDetailsUrlParam.toSearchString({ value: details });
+export function getDetailsUrl(selfLink: string, resetSelected = false, mergeGlobals = true) {
```

Comment:
Function get much harder to understand with this triadic arguments. Is it possible to avoid `mergeGlobals` at least. And it's always `true` as I see.

Reply:
Yes, currently it's always `true`. 
I would prefer to extend the function as `getDetailsUrl(selfLink: string, {resetSelected = false, mergeGlobals = true} = {})` but it would be _breaking change_ since it's exported to extensions-api.

Reply:
It is not necessarily a breaking change since you could do `getDetailsUrl(selfLink: string, opts: boolean | DetailsUrlOpts = false)`